src/components/navBar/navBar.js

```diff
@@ -461,9 +461,9 @@ function MdNavItemController($element) {
   /** @private {boolean} */
   this._focused = false;
 
-  var hasNavClick = this.mdNavClick != null;
-  var hasNavHref = this.mdNavHref != null;
-  var hasNavSref = this.mdNavSref != null;
+  var hasNavClick = !!($element.attr('md-nav-click'));
```

Comment:
the actual values aren't boolean; I just need to know if they have been provided at all - so just checking for truthiness is ok.